### ISSUE TITLE
MACTAP needs edits

### ISSUE THREAD

**Issue by larsbrinkhoff** [PDP-10/its#1607]
MACTAP doesn't build as is.  It needs some minor edits to go well with MIDAS.

**Comment by larsbrinkhoff** [comment]
Error messages:

```
LOOK+4          GL+77    0.    12-013   LOOK1   Label inside <>, () or []
LOOK1           GL+77    0.    12-016   POPJ1A  Label inside <>, () or []
POPJ1A          GL+77    2.    12-019   LOC     Within <>, () or []
POPJ1A          GL+77    2.    12-019   LOC     Within <>, () or []
POPJ1A          GL+77    0.    12-023   OPNRD   Label inside <>, () or []
OPNRD           GL+77    0.    12-056   SAVAC0  Label inside <>, () or []
SAVAC0          GL+77    0.    12-057   SAVAC   Label inside <>, () or []
Within groupings: [ at 12-058, [ at 12-043, [ at 12-010, [ at 12-005
LOOK+4          GL+77    0.    12-058   PDL overflow

LOOK+4          GL+77    0.    12-058   Stray )
LOOK+4          GL+77    0.    12-058   Stray ]
FILE1+5         GL+15    0.    13-023   FILE1B  Label inside <>, () or []
FILE1B          GL+15    0.    13-028   FILE3   Label inside <>, () or []
FILE3           GL+15    0.    13-032   FILE4   Label inside <>, () or []
FILE4           GL+15    0.    13-041   FILE5   Label inside <>, () or []
FILE5           GL+15    0.    13-045   FILE7   Label inside <>, () or []
FILE7           GL+15    0.    13-047   Formfeed within <>, () or []
FILE7           GL+15    0.    14-001   OPNWR   Label inside <>, () or []
OPNWR           GL+15    0.    14-013   FILE6   Label inside <>, () or []
FILE6           GL+15    0.    14-029   FILER   Label inside <>, () or []
FILER           GL+15    2.    14-046   LOC     Within <>, () or []
Within groupings: [ at 13-023, [ at 13-022
FILE1+5         GL+15    2.    14-046   PDL overflow

FILE1+5         GL+15    2.    14-046   Stray )
FILEST+15       GL+50    0.    16-017   ERR3    Label inside <>, () or []
ERR3            GL+50    0.    16-018   ERR4    Label inside <>, () or []
ERR4            GL+50    0.    16-021   DELE    Label inside <>, () or []
DELE            GL+50    0.    16-023   DELE2   Label inside <>, () or []
DELE2           GL+50    0.    16-035   FILESL  Label inside <>, () or []
FILESL          GL+50    0.    16-049   UDELE   Label inside <>, () or []
UDELE           GL+50    0.    16-052   Formfeed within <>, () or []
UDELE           GL+50    0.    17-001   URED1   Label inside <>, () or []
URED1           GL+50    0.    17-003   URED    Label inside <>, () or []
URED            GL+50    0.    17-006   UREDT   Label inside <>, () or []
UREDT           GL+50    0.    17-022   URED3   Label inside <>, () or []
URED3           GL+50    0.    17-028   UREDE   Label inside <>, () or []
UREDE           GL+50    2.    17-031   LOC     Within <>, () or []
UREDE           GL+50    2.    17-031   LOC     Within <>, () or []
UREDE           GL+50    0.    17-033   URED2   Label inside <>, () or []
Within groupings: [ at 17-028, [ at 16-014
FILEST+15       GL+50    0.    17-033   PDL overflow

**Comment by larsbrinkhoff** [comment]
This is mostly unterminated `[` literals.

**Comment by larsbrinkhoff** [comment]
Fixed in #1630.
